web/src/components/map/box.js

```diff
@@ -16,7 +22,7 @@ mapboxgl.workerClass = require("worker-loader!mapbox-gl/dist/mapbox-gl-csp-worke
  * @property {Array<Pollutant>} pollutants
  */
 export const MapBox = ({ isLoading, pollutants }) => {
-  const mapStyle = "mapbox://styles/mapbox/streets-v11";
+  const mapStyle = process.env.REACT_APP_MAPBOX_STYLE_URL;
```

Comment:
This might be better as a constant saved in the `./map/utils.js` file. There is nothing secret about it and we should want it to be the same, regardless of environment.